### PR TITLE
Feature/structure

### PR DESCRIPTION
**Pull request by tnyandoro**
Learning objectives
Use webpack to bundle JavaScript.
Learn how to use proper ES6 syntax.
Use ES6 modules to write modular JavaScript.
Estimated time: 3.5h
Description
In this project, you will build a simple HTML list of To Do tasks. The list will be styled according to the specifications listed later in this lesson. This simple web page will be built using webpack and served by a webpack dev server.

IMPORTANT NOTE: Read all requirements before you start building your project.

General requirements
Make sure that there are no linter errors.
Make sure that you used correct GitHub Flow.
Make sure that you documented your work in a professional way.
HTML/CSS requirements
Follow our list of the best practices for HTML & CSS.
JavaScript requirements
Follow our list of best practices for JavaScript.
Project requirements
Set up a new project with webpack that is based on the webpack exercise you have already completed.
Create an index.html file and write your HTML markup here. Create an empty To-Do List placeholder (<div> or <ul> element). The index.html file must be set as a template using the HTML Webpack Plugin.
Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
description [string].
completed [bool].
index: [number].
Write a function to iterate over the tasks array and populate an HTML list item element for each task.
On the page, load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
Create a style.css and set rules for the To Do List. CS

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
+/* eslint-disable func-names */
+// import css from "./style.css";
+// declarations
+const form = document.querySelector('#itemForm'); // select the form
+const itemInput = document.querySelector('#itemInput'); // select the input box from the form
+const itemList = document.querySelector('.item-list');
+const inform = document.querySelector('.inform');
+const clearButton = document.querySelector('#clear-list');
+
+let todoItems = [];
```
Comment (smcommits) [inline src/index.js:13]: - [ ] **In this milestone of the project, you are required to create an array containing simple to-do tasks. Please implement it.**

You have to create an array of some simple to-do tasks (array of objects) with each object containing three keys - description [string], completed [bool], index: [number]. However, you are free to have more properties or name them differently. 

Please make sure to dynamically append the predefined tasks into DOM.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "the-todo-list",
+  "version": "1.0.0",
+  "description": "In this project, we have build a simple HTML list of To Do tasks. The list will be styled according to the specifications listed later in this lesson. This simple web page will be built using webpack and served by a webpack dev server.",
+  "private": "true",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
```
Comment (smcommits) [inline package.json:9]: - [ ] **When I run this script, the HTML document is not loaded. Please fix this**

Upon running `npm run start` I am getting a black document with no elements. For some reason, your HTML document is not loaded. Please fix this.

**Review by smcommits — CHANGES_REQUESTED** [review]
Hi @tnyandoro, 

### You have done an excellent job. The project appears to be fantastic. It reflects great efforts and good understanding. 
### To make this project even better, there are some issues that you still need to work on. You are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


 ## Required Changes ♻️
_Please check the inline comments below._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tnyandoro** [comment]
in this milestone of the project, you are required to create an array containing simple to-do tasks. Implemented,
When I run this script, the HTML document is not loaded. Please fixed

File: dist/index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"
+    integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="stylesheet" href="57011448dc08b9372b5e.css">
+<script defer src="main.js"></script></head>
```
Comment (carlos-ssh) [inline dist/index.html:17]: - Kindly add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @tnyandoro , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tnyandoro** [comment]
Added the Readme

**Review by carlos-ssh — APPROVED** [review]
Hi @tnyandoro , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**